web-api/terraform/bin/import-api-public-stages.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+# This script is temporary, and can be removed once all environments (Court, Flexion)
+# have their Public API stages imported.
```

Comment:
Importing the existing Public API stages works great — though this script is a bit of a beast. We can delete it once the import is completed — added a TODO to #711.